pkg/planner/core/explain_ru_test.go

```
@@ -63,27 +64,189 @@ func TestExplainAnalyzeRUFormat(t *testing.T) {
+			require.NoError(tb, err)
+			return ru
+		}
+		require.FailNowf(tb, "operator not found", "operator %s not found in rows %v", operator, rows)
+		return 0
+	}
+	insertIntRows := func(table string, start, count int) {
+		var sql strings.Builder
+		sql.WriteString("insert into ")
+		sql.WriteString(table)
+		sql.WriteString(" values ")
+		for i := 0; i < count; i++ {
```

golangci-lint (2.12.2)

[error] for loop can be changed to use an integer range (Go 1.22+)

(intrange)